internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -0,0 +1,89 @@
+package tart
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/cirruslabs/cirrus-cli/internal/executor/instance/persistentworker/remoteagent"
+	"github.com/cirruslabs/cirrus-cli/internal/executor/instance/runconfig"
+	"github.com/cirruslabs/cirrus-cli/internal/executor/platform"
+	"github.com/cirruslabs/cirrus-cli/internal/logger"
+	"time"
+)
+
+var (
+	ErrFailed = errors.New("tart isolation failed")
+)
+
+type Tart struct {
+	logger      logger.Lightweight
+	vmName      string
+	sshUser     string
+	sshPassword string
+}
+
+func New(vmName string, sshUser string, sshPassword string, opts ...Option) (*Tart, error) {
+	parallels := &Tart{
```

Comment:
should it be called `tart`?

Reply:
Indeed, fixed in https://github.com/cirruslabs/cirrus-cli/pull/493/commits/f477004225562119a49ef3349d24ecfbc26353a0.